randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1509,6 +1509,13 @@ Extra - asset_id: collision_camera_024
           Ice Missile and Movement (Intermediate) and Use Spin Boost
   > Door to Energy Recharge Station
       Trivial
+  > Pickup (Missile Tank)
+      All of the following:
+          Morph Ball and Simple IBJ
+          Any of the following:
+              Space Jump
+              Flash Shift and Spin Boost and Walljump (Advanced)
+              Speed Booster and Movement (Intermediate)
```

Comment:
If Simple IBJ is used, then there is no reason for any of the paths that follow. I feel like this is a mistake.